ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -212,7 +212,7 @@ def test005_restore_archived_orders(self):
             self.assertTrue(self.order_page.is_order_exist(
                 value=selected_order_data['Analysis No.']))
 
-    # will continue with us 
+    # will continue with us
     def test006_deleted_archived_order(self):
```

Comment:
There is another implementation for this test case in another [PR](https://github.com/modeso-open-source/1lims-automation/pull/167/files#diff-05c34fcc59d5beb73a7b86f1cf41a0c1R218). could u check both of them?

Reply:
yes, will check

Reply:
@0xIslamTaha you have two concepts regarding the delete approach here that we can delete the main order or the sub order with two separate test cases so I updated the main to be more clear that test case regarding delete the main order ( test 006 )

Reply:
@dianaeldemerdash I didnt get the point